odl/tomo/backends/skimage_radon.py

```diff
@@ -90,9 +93,12 @@ def skimage_radon_forward_projector(volume, geometry, proj_space, out=None):
     if out is None:
```

Comment:
Here you still use the `if out is None:`-construction. What is the difference with `none_context` used above?

Reply:
Right, I could use it here, too.

Reply:
There's a slight difference, in that the interpolator returns a NumPy array, not a ``proj_space`` element, so we can do either this, or call `proj_space.element` on the result. I'll stick with this version for now.